Makefile

```diff
@@ -4,6 +4,14 @@ PACKAGE_NAME:=github.com/maistra/istio-workspace
 OPERATOR_NAMESPACE?=istio-workspace-operator
 TEST_NAMESPACE?=bookinfo
 
+ifeq ($(shell uname), Linux)
+PLATFORM = linux-gnu
+else
+PLATFORM = apple-darwin
```

Comment:
This is for `operator-sdk` - wondering if deserves a comment. WDYT?

Reply:
PLATFORM is hardcoded in a similar way in the operators-sdk scripts?

Reply:
any reason to align our .goreleaser.yml naming?

Reply:
The `PLATFORM` here is for `operator-sdk` download only. Maybe it deserves a better name or even being defined in the target itself.

> PLATFORM is hardcoded in a similar way in the operators-sdk scripts?

I am not sure what you mean by scripts here.

> any reason to align our .goreleaser.yml naming?

No :) I can understand the reasoning that it's easier to understand `apple-darwin` than just `Darwin`, but it makes downloading/automation harder because it doesn't match output from `uname`.

```
$ uname -s
Linux

$ uname -p
x86_64
```

This way your download link simply becomes `ike_${VERSION}_$(uname -s)_$(uname -p).tar.gz`

Reply:
operator-sdk scripts=their release process. As to why/how they possible ended up with that naming. :)

Reply:
Not sure, but I will investigate.

![investigation-fbi](https://user-images.githubusercontent.com/719616/61043398-73a47f80-a3d6-11e9-97e9-00b2a6ee9642.gif)

Reply:
18525bf